### ISSUE TITLE
Input should blur() on escape

### ISSUE THREAD

**Issue by kashav** [vector-im/element-web#16878]
#### Is your suggestion related to a problem? Please describe.

I don't want others to see a "kashav is typing..." message if I'm not typing anything.

#### Describe the solution you'd like.

`esc` should blur() the input

**Comment by kashav** [comment]
I can take a stab at this if there's interest.

**Comment by SimonBrandner** [comment]
@kashav, yeah that would be great!

**Comment by SimonBrandner** [comment]
> I don't want others to see a "kashav is typing..." message if I'm not typing anything.

I _think_ the typing notification will actually disappear if you stop typing - no need to blur()

Still, this solution would make Element more accessible, so please go ahead if you want!

**Comment by SimonBrandner** [comment]
Ah, you're right. This behaviour made sense in my head but doesn't seem to be the standard. Is this still an issue then?

**Comment by kashav** [comment]
> I _think_ the typing notification will actually disappear if you stop typing - no need to blur()

Neat, was not aware of this. I believe visible confirmation of this would be helpful.

> Testing with this standard textarea in github where I am typing this message Escape does certainly not blur the field. (tested on other textareas with same behaviour)

Maybe this is platform-dependant. In Firefox on Fedora, escape blurs the Github textarea (as well as the textarea example on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea), but yeah, not Slack.

My workaround has been to click the whitespace above the composer, and I don't mind continuing to do that.
